### PR TITLE
added ShubhamRattra

### PR DESCRIPTION
**Pull request by ShubhamRattra**
added Shubham Rattra to take a pledge.

### CODE REVIEW

File: src/data/contributors/ShubhamRattra.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Shubham Rattra",
+    "skills" : ["Adroid development", "Java", "Kotlin"],
+    "cause" : "I code for the community",
```
Comment (KeenWarrior) [inline src/data/contributors/ShubhamRattra.json:4]: **I code for** is already part of string. 

Additionally, please write something unique to you as cause.

File: src/data/contributors/ShubhamRattra.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    "image" : "Avatar1",
+    "github" : "https://github.com/ShubhamRattra",
+    "twitter" : "https://twitter.com/ShubhamRattra",
+    "linkedin" : "https://www.linkedin.com/in/shubham-rattra-37b57a179/"
+}
```
Comment (KeenWarrior) [inline src/data/contributors/ShubhamRattra.json:9]: new line at end of file is a good rule of thumb to follow.

File: src/data/contributors/ShubhamRattra.json (hunk 1 above)
Comment (ShubhamRattra) [reply src/data/contributors/ShubhamRattra.json:4]: I have written the community. Or do I have to write something else?

Comment (KeenWarrior) [reply src/data/contributors/ShubhamRattra.json:4]: You can write anything. you can take hint from the readme.

**Comment by KeenWarrior** [comment]
@ShubhamRattra Please remove **I code for** text. As it is part of HTML behind and not needed.

**Pull request merged**